src/main/java/com/yscope/logging/log4j1/AbstractBufferedRollingFileAppender.java

```diff
@@ -504,11 +513,13 @@ private class SyncRequest extends Request {
       public final String logFileBaseName;
       public final long logRolloverTimestamp;
       public final boolean deleteFile;
+      public final Map<String, Object> metadata;
```

Comment:
You're not using this anywhere?

Reply:
There are currently no concrete classes in this library that uses the metadata. However, downstream concrete appenders extending the abstract class will use them.

Reply:
How are they going to access them?

Reply:
Good point. Made changes to `sync` method to allow downstream appenders to use it.